### PR TITLE
Added new maven module for the Notice.txt template file to be used in gateway

### PR DESCRIPTION
**Pull request by mgherghe**

### CODE REVIEW

**Comment by dpwspoon** [comment]
This looks ok to me

File: community.notice/src/main/resources/org/kaazing/community/notice/third-party-template.ftl [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <#list licenseMap as projList>
+            <#list projList.getValue() as proj>
+                <#if proj == p>
+                    <#assign mvnLicense = mvnLicense + projList.getKey() + "\n"/>
+                </#if>
+            </#list>
+        </#list>
+    <#else>
+        <#list p.getLicenses() as license>
+            <#assign mvnLicense = mvnLicense + (license.url!"null") + " (" + license.name + ")\n"/>
```
Comment (dpwspoon) [inline community.notice/src/main/resources/org/kaazing/community/notice/third-party-template.ftl]: what happens if other things don't exist like license? Can we give hints?

File: community.notice/src/main/resources/org/kaazing/community/notice/third-party-template.ftl [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        </#list>
+    </#if>
+    <#assign homepage = "\tHomepage:\t" + (p.url!"Home page is not included in maven artifact, and thus couldn't be referenced here") + "\n"/>
```
Comment (mgherghe) [inline community.notice/src/main/resources/org/kaazing/community/notice/third-party-template.ftl]: I didn't see any option for configuring hints for the homepage. The text was from the previous license plugin, which displayed: 
>Home page is not included in maven artifact, and thus couldn't be referenced here

Now, if no homepage is found, it displays:
>	Homepage:	Home page is not included in maven artifact, and thus couldn't be referenced here

Should any of the two options be displayed, or should nothing be listed if there is no url for the homepage?

Comment (dpwspoon) [inline community.notice/src/main/resources/org/kaazing/community/notice/third-party-template.ftl]: Please change the text to

"The project URL is not included in this dependency's pom, and thus could not be referenced here."

Consider filing a improvement request to the license projects issue tracker to add it as a hint?

**Comment by dpwspoon** [comment]
LGTM.  @adrian-galbenus / @AdrianCozma  can you do a release

**Comment by AdrianCozma** [comment]
@dpwspoon I tried a release but it failed to connect to maven central:
```
[ERROR] Failed to execute goal org.sonatype.plugins:nexus-staging-maven-plugin:1.6.2:deploy (injected-nexus-deploy) on project community.parent: Execution injected-nexus-deploy of goal org.sonatype.plugins:nexus-staging-maven-plugin:1.6.2:deploy failed: Nexus connection problem to URL [https://oss.sonatype.org/ ]: 401 - Unauthorized -> [Help 1]
```

**Comment by AdrianCozma** [comment]
@dpwspoon the credentials seem to be outdated, I can't even login to the sonatype interface, [https://oss.sonatype.org/. I used the credentials from passpack, sonatype release credentials.

**Comment by dpwspoon** [comment]
@adrian-galbenus I believe this has been resolved, and @mgherghe needs a release.  Please try again.

**Comment by mgherghe** [comment]
This was released by @AdrianCozma with version 2.24.

**Pull request merged**